contracts/adapters/Configuration.sol

```diff
@@ -0,0 +1,115 @@
+pragma solidity ^0.7.0;
+
+// SPDX-License-Identifier: MIT
+
+import "../core/DaoConstants.sol";
+import "../core/DaoRegistry.sol";
+import "../guards/MemberGuard.sol";
+import "./interfaces/IConfiguration.sol";
+import "../utils/SafeMath.sol";
+import "../adapters/interfaces/IVoting.sol";
+
+/**
+MIT License
+
+Copyright (c) 2020 Openlaw
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+ */
+
+contract ConfigurationContract is IConfiguration, DaoConstants, MemberGuard {
+    using SafeMath for uint256;
+    enum ConfigurationStatus {IN_PROGRESS, DONE}
+
+    struct Configuration {
+        ConfigurationStatus status;
+        bytes32[] keys;
+        uint256[] values;
+        bool exists;
```

Comment:
instead of having an "exists" field, just add an enum and call it "NOT_CREATED" and by default that would be the value.

This way you save a storage which is relatively expensive

Reply:
Sure, will do.  I modeled this after the ragekick adapter so the same change might apply there.

Reply:
I made the same comment to Felipe on ragequit. It has already changed